tests/test-runner/bin/test-runner.py.in

```diff
@@ -269,6 +270,14 @@ User: %s
         except OSError as e:
             fail('%s' % e)
 
+        """
+        Log each test we run to /dev/kmsg (on Linux), so if there's a kernel
+        warning we'll be able to match it up to a particular test.
+        """
+        if exists("/dev/kmsg"):
+            cmd = "sudo sh -c 'echo ZTS run %s > /dev/kmsg'" % self.pathname
+            os.system(cmd)
```

Comment:
While I don't think this is a bad idea in-of itself, I'm not a fan of this approach since sudo *already* syslogs its argv, so this should end up in the journal all the same already, and this'd double-log (and, if you're on the console, this would render test output absolutely unusable, right?).

Also, you've missed `SUDO` and are using `os.system()`, so either this'd be much better written as
```python
		try:
            kp = Popen([SUDO, "sh", "-c", f"echo ZTS run {self.pathname} > /dev/kmsg"])
            kp.wait()
        except Exception:
            pass
```